indexing-service/src/test/java/io/druid/indexing/common/task/CompactionTaskTest.java

```diff
@@ -109,13 +109,13 @@
   private static final Interval COMPACTION_INTERVAL = Intervals.of("2017-01-01/2017-06-01");
   private static final Map<Interval, DimensionSchema> MIXED_TYPE_COLUMN_MAP = ImmutableMap.of(
       Intervals.of("2017-01-01/2017-02-01"),
-      new StringDimensionSchema(MIXED_TYPE_COLUMN, null),
+      new StringDimensionSchema(MIXED_TYPE_COLUMN, null, null),
```

Comment:
minor nit - StringDimensionSchema has a constr for just name, we can use that here.

Reply:
Sure, I changed these.